src/dcat-ap/dcat-dataset.ts

```diff
@@ -33,9 +37,16 @@ export const defaultCalculatedFields = [
   'provenance'
 ];
 
-export function getDcatDataset(hubDataset: any, orgBaseUrl: string, orgTitle: string, siteUrl: string) {
+export function getDcatDataset(hubDataset: any, orgBaseUrl: string, orgTitle: string, siteUrl: string, siteModel: IModel) {
```

Comment:
It's fine for now, but just note that 5 function parameters is pretty unwieldy.

Reply:
I know, it's awful. I'm hoping we can clean things up when we try an unify logic between the two plugins.